docs/server.rst

```diff
@@ -36,12 +36,54 @@ Prerequisites
 -------------
 
 The first thing you need is a recent (>= 2.6.36) Linux kernel that supports L2TPv3
-tunnels. The following modules are required for Tunneldigger operation:
+tunnels.
+
+These distributions are supported:
+
+* Debian
+* Fedora with the package ``kernel-modules-extra``
+* *add your distribution*
+
+Kernel Modules
+~~~~~~~~~~~~~~
+
+The following modules are required for Tunneldigger operation:
 
 * ``l2tp_core``
 * ``l2tp_eth``
 * ``l2tp_netlink``
 
+You can find out if these modules are loaded by running ``lsmod | grep l2tp``.
+If you get no output, they need to be activated.
+For the activation of the kernel modules, we recommend adding a file
+``/etc/modules-load.d/tunneldigger.conf`` with the following content:
```

Comment:
People should just always add that. Just because `lsmod` lists these now, doesn't mean they will still be there after a reboot or after some unrelated configuration changed.

Reply:
4cddd02 restructures this and puts the activation first